model.py

```diff
@@ -4990,6 +4990,7 @@ class CachedFeed(Base):
     GROUPS_TYPE = 'groups'
     PAGE_TYPE = 'page'
     RECOMMENDATIONS_TYPE = 'recommendations'
+    SERIES_TYPE = 'series'
```

Comment:
However, the grouped feed will link to individual page-type feeds, and those feeds will need different types, so leave this here.